### PR TITLE
Adding HST public data

### PR DESCRIPTION
**Pull request by arfon**
This is a work in progress. Opening this up to get some eyes on this from @ivastar.

### CODE REVIEW

File: datasets/hst.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Name: HST public data
+Description: |
+  The Hubble Space Telescope (HST) is one of the most productive scientific instruments ever created. This dataset contains calibrated and raw data for all of the currently active instruments on HST: ACS, COS, STIS and WFC3. 
+Documentation: http://astroquery.readthedocs.io/en/latest/mast/mast.html
+Contact: archive@stsci.edu
+UpdateFrequency: Hourly
+Tags:
```
Comment (arfon) [inline datasets/hst.yaml]: @jedsundwall is there any guidance on what tags we should use here?

File: datasets/hst.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  - hst
+  - hubble
+  - nasa
+License: STScI herby grants the non-exclusive, royalty free, non-transferable, worldwide right and license to use, reproduce and publicly display in all media public data from the Hubble Space Telescope.
+Resources:
+  - Description: Project data files
+    ARN: arn:aws:s3:::stpubdata
+    Region: us-east-1
+    Type: S3 Bucket
+  - Description: Notifications for new data
```
Comment (arfon) [inline datasets/hst.yaml]: I don't think we have notifications set up on this bucket? Or perhaps we do and I just don't know? / cc @mfox22

File: datasets/hst.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    ARN: arn:aws:sns:us-east-1:928094251383:gdelt-csv
+    Region: us-east-1
+    Type: SNS Topic
+DataAtWork:
+  - Title: Making HST Public Data Available on AWS
+    URL: http://mast-labs.stsci.io
```
Comment (arfon) [inline datasets/hst.yaml]: This isn't public yet. Also, I think we might want to cross-post on the MAST blog as well as the AWS blog. Will follow up with email to @jedsundwall

File: datasets/hst.yaml (hunk 1 above)
Comment (jflasher) [reply datasets/hst.yaml]: Hi @arfon, you can see blessed tags at https://github.com/awslabs/open-data-registry/blob/master/tags.yaml. If you really think one should be added that's not in there, let us know. We're trying to strike a balance between being helpful but not overly descriptive.

File: datasets/hst.yaml (hunk 2 above)
Comment (jflasher) [reply datasets/hst.yaml]: Some instructions on how to set this up at https://gist.github.com/jedsundwall/7b5ea0a33cc3ca0b9764f7090a59858a. Would be great to get this in place, if you need any help, just let me know.

File: datasets/hst.yaml (hunk 1 above)
Comment (ivastar) [reply datasets/hst.yaml]: @jflasher Could "astronomy" be added to the tags? Biology, music, genomics, etc. are in the tags. This does not fit in "earth observations".

Comment (jflasher) [reply datasets/hst.yaml]: Yep, I think that's fine. If you just want to add the tag to tags.yaml and include it in this PR, that'd work.

Comment (arfon) [reply datasets/hst.yaml]: 👍done in https://github.com/spacetelescope/open-data-registry/commit/af7dff3a59756565f6e24a058bc7077707693d14

**Comment by arfon** [comment]
👋 @jedsundwall & @jflasher - I think this PR is ready to roll. We're planning on announcing the availability of this dataset tomorrow in time for the AAS summer meeting in Denver this week. 

Could this be merged?

File: datasets/hst.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+Description: |
+  The Hubble Space Telescope (HST) is one of the most productive scientific instruments ever created. This dataset contains calibrated and raw data for all of the currently active instruments on HST: ACS, COS, STIS and WFC3. 
+Documentation: http://astroquery.readthedocs.io/en/latest/mast/mast.html
+Contact: archive@stsci.edu
+UpdateFrequency: Hourly
+Tags:
+  - astronomy
+License: STScI herby grants the non-exclusive, royalty free, non-transferable, worldwide right and license to use, reproduce and publicly display in all media public data from the Hubble Space Telescope.
+Resources:
+  - Description: Project data files
```
Comment (jflasher) [inline datasets/hst.yaml]: Could you make a note that this is a Requester Pays bucket so people know to add the flag as appropriate? Other than that, looks good to go!

**Comment by arfon** [comment]
> One requested text change, then should be good to merge.

Done: https://github.com/awslabs/open-data-registry/pull/45/commits/e48cdcf8668f082eed7b42875977a13abed44c01

**Comment by jflasher** [comment]
Merged, will be live in a few moments. Thanks!

**Pull request merged**